### ISSUE TITLE
VsVim is being injected into R#'s unit test "output" panel

### ISSUE THREAD

**Issue by Porges** [jaredpar/VsVim#1720]
In R#'s "Unit Test Sessions" window, VsVim is in the "output" panel. (This is basically the same thing as in #1498.)

This seems to be causing something pathological to occur as I have a test that writes a lot to the output window and VS is hanging.

It's not a very useful place to be as the "output" is read-only :)

R# version: 10.0.1
VsVim: 2.0.1.0
VS: 14.0.42720.0

**Comment by jaredpar** [comment]
Thanks for reporting.  I will take a look at this. 

I agree it's not very useful.  Unfortunately the way Visual Studio extensibility works extensions essentially have to opt-out of windows.  Hence whenever a new window shows up like this I need to go and change the detection logic to recognize it's a window VsVim shouldn't be inside of and stop inserting into it.

**Comment by jaredpar** [comment]
I just tried to repro this on R# 9.0 and could not do so.  I created a couple of tests, had them fail by throwing an unhandled exception.  This caused some text to appear in the output window.  VsVim was not installed there.  

It's possible this is just a R# 10.0 issue though.  I'm doing a pass on 9.0 right now to take care of some bugs.  Once those are done I'm going to upgrade to 10.0 and I'll try this bug out again.
